test/functional/1-telemetry.js

```diff
@@ -20,6 +20,7 @@ describe("telemetry", function() {
     driver = await utils.setupWebdriver.promiseSetupDriver(
       utils.FIREFOX_PREFERENCES,
     );
+    await utils.setPreference(driver, "extensions.fastblock-shield_mozilla_org.test.variationName", "TPL0");
```

Comment:
I don't like that this has to be set, I'm getting failures. If I set it to "FB5L0" there's a failure on "correctly records performance metrics"
on "Control" there's a failure on "has recorded one ping"
Doesn't this sort of invalidate the tests?

Since this branch is needed in order to record amount of trackers on a page, couldn't we just set this before running that test in particular.

Reply:
shoot, I think the errors are not your fault, I'm seeing them on master too - but only with certain branches so it's unpredictable. Sometimes I get a positive test, other failing

Reply:
I replaced it with our well-working default test site: https://itisatrap.org/firefox/its-a-tracker.html